tests/storage/export.go

```diff
@@ -297,6 +297,7 @@ var _ = Describe(SIG("Export", func() {
 
 		pod, err := createSourcePodChecker(pvc)
 		Expect(err).ToNot(HaveOccurred())
+		Eventually(ThisPod(pod), 30*time.Second, 1*time.Second).Should(HaveConditionTrue(k8sv1.PodReady))
```

Comment:
nit
```suggestion
		Eventually(ThisPod(pod)).WithTimeout(30*time.Second).WithPolling(1*time.Second).Should(HaveConditionTrue(k8sv1.PodReady))

```

Reply:
Don't we need to check if the container is running rather then the pod to be ready?

Reply:
Ready=True is a stronger condition which indicates something it's 100% operational
https://maelvls.dev/kubernetes-conditions/

Reply:
> I prefer to stick with this 🙏

Can you please explain why? :)

Reply:
Unless we provide a solid, agreed upon standard then I don't see the value in changing one occurrence in this file